src/encoded/schemas/user.json

```diff
@@ -94,7 +94,8 @@
         }
     },
     "boost_values": {
-        "title": 1.0
+        "title": 1.0,
+        "viewing_groups": 1.0
```

Comment:
What does this change do?

Reply:
That was likely a left-over from a test. Removed